src/.formatted-files

```diff
@@ -1,34 +1,36 @@
-stat.h
```

Comment:
So they are sorted.  When I make a patch I need to ensure everything is clang-formatted.  At present it's hard to know if a given file is clang-formatted because they are in a random order.